bulk-import/ndjson-generator/Helpers/DataGenerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace NDJsonGenerator.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class DataGenerator
```

Comment:
Consider sealing the class, and making it internal, unless you intend on this being specialized.

Reply:
Making it static